src/test/cli/rbd/invalid-snap-usage.t

```diff
@@ -89,6 +89,7 @@
   rbd: snapname specified for a command that doesn't use it
   [22]
   $ rbd bench-write foo@snap
+  rbd: bench-write is deprecated, use rbd bench --io-type write ...
```

Comment:
@yangdongsheng Udate this command too? I think it is better to remove bench-write usage everywhere.

Also, looking at this error, right now `bench --io-type read` returns this too, but I think we can allow bench read for snapshots?